packages/functions/lambdas/dbOperations.ts

```diff
@@ -0,0 +1,28 @@
+import { executeSql } from "@/db/queries/dbOperations";
+import { errorResult, okResult } from "@/utils/lambda/result";
+
+export const handler = async (event: { sql: string }) => {
+  console.log("🚀 ~ handler ~ event:", event);
+
+  try {
+    // Extract tableName from the event object
+    const { sql } = event;
+
+    if (!sql) {
+      throw new Error("sql parameter is required");
+    }
+
+    console.log("Cleaning up db...");
```

Comment:
How do we know its a cleanup?

Reply:
It was a cleanup before i made it generic SQL executer, removing.